src/components/calculator/util/budgetUtil.ts

```diff
@@ -1,31 +1,59 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
 import {
   fixedPointPrecision,
   fixedPointPrecisionPercent,
 } from './FormatPrecision'
 
+export type BudgetConsumptionInfo = {
+  translateString: string
+  value: { multiple?: string; percentage?: string }
+}
+
 export const getWeekBudget = (budget: number): number => {
   return budget / 50 // Numbers look cleaner than 52.
 }
 
-export const budgetConsumption = (
+export const getBudgetConsumptionInfo = (
   points: number,
   budget: number,
-  multiple_suffix: string,
-  percentage_suffix: string,
-): string => {
+): BudgetConsumptionInfo => {
   const displayedPoints = Number(
     fixedPointPrecision(
       points,
       /*maxNumber=*/ undefined,
-      /*decimalsNearMax=*/ undefined,
-      /*thousandsSeparator=*/ '',
+      /*minNumber=*/ undefined,
```

Comment:
Could we use an object parameter instead? https://exploringjs.com/impatient-js/ch_callables.html#named-parameters

Reply:
done